src/models/exchange.ts

```diff
@@ -41,10 +41,29 @@ export function solveTimeRemaining(
   return [Math.max(remaining - FINALITY_DURATION, 0), remaining];
 }
 
-const GP_GRAPH = "https://api.thegraph.com/subgraphs/name/gnosis/protocol";
+export enum Network {
+  Mainnet = "mainnet",
+  Rinkeby = "rinkeby",
+  Xdai = "xdai",
+}
+
+const GP_GRAPH: Record<Network, string | undefined> = {
```

Comment:
Because all possible values of `Network` are set, maybe this could be `Record<Network, string>`?